insights/core/__init__.py

```diff
@@ -14,7 +14,7 @@
 
 from insights.contrib.ConfigParser import NoOptionError, NoSectionError
 from insights.core import ls_parser
-from insights.core.exceptions import ContentException, ParseException, SkipComponent, SkipException  # noqa: F401
```

Comment:
Please keep it there, there may be other apps importing the SkipException from here.